.eslintrc

```diff
@@ -1,6 +1,37 @@
 {
-  "extends": "canonical",
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint",
+    "import"
+  ],
+  "extends": [
+    "canonical",
+    "eslint:recommended",
```

Comment:
Please use `canonical/typescript`.

Reply:
Sorry, I mean, `canonical` + `canonical/typescript`. The rest are not needed.

Reply:
Okay. I will fix that.